test/cookbooks/test/recipes/extension.rb

```diff
@@ -1,28 +1,37 @@
 postgresql_repository 'install'
+# # Dokken images don't have all locales available so this is a workaround
+locale = case node['platform_family']
+         when /^(debian|ubuntu|fedora)$/
+           'C.UTF-8'
+         when 'rhel'
+           if node['hostname'].match?(/oracle/)
+             'C.UTF-8'
+           else
+             node['platform_version'].to_i < 8 ? 'en_GB.UTF-8' : 'C.UTF-8'
+           end
+         when 'oracle'
+           'C.UTF-8'
+         else
+           'en_US.UTF-8'
+         end
```

Comment:
Why couldn't you keep the same form as below?

Reply:
It didnt work for some OSs, I dont really understand locales. Oracle doesnt Identify itself, and needs the same locale as centos8 but  7 is different idk, Ill try the old way again but I think this is the only way I found that works. maybe you could help me find a more elegant solution

Reply:
I'll take a closer look at this.